content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

```diff
@@ -417,11 +417,11 @@ Type: `string`
 
 Values: `str`, `email`, `int64`, `int32`, `bytes`, `int32_string`, `int64_string`
 
-Depends on: `tokenized`
+Depends on: `tokenized` (before 0.93.0)
 
 Group: `tokenization`
 
-Description: configures type of [tokens](/acra/security-controls/tokenization/). Works only together with `tokenized` option. 
+Description: configures type of [tokens](/acra/security-controls/tokenization/). Works only together with `tokenized` option (before 0.93.0). 
```

Comment:
```suggestion
Description: configures type of [tokens](/acra/security-controls/tokenization/). Configure a data type to enable tokenization for a column, Acra will create tokens with specified data type. Before Acra 0.93.0 `token_type` worked only if `tokenized` option was enabled. Since 0.93.0, `token_type` doesn't require `tokenized` flag at all to simplify the config file.
```